app/src/main/res/values/strings.xml

```diff
@@ -100,21 +100,21 @@
     <string name="enable_playback_state_lists_summary">Show playback position indicators in lists</string>
     <string name="settings_category_clear_data_title">Clear data</string>
     <string name="enable_watch_history_summary">Keep track of watched videos</string>
-    <string name="resume_on_audio_focus_gain_title">Resume on focus gain</string>
-    <string name="resume_on_audio_focus_gain_summary">Continue playing after interruptions (e.g. phone calls)</string>
+    <string name="resume_on_audio_focus_gain_title">Resumed playing</string>
```

Comment:
I think this should be reverted. It is missing the "focus gain" information

Reply:
I prefer @comradekingu's change because I do not think that the "focus gain" info is valuable for the user. What is the difference between "Resume on focus gain" and "Resumed playing" form a user's viewpoint? I think "resumed playing" is a better title for the setting because it tells you what the setting is about: resume a paused video or audio. The info when this is triggered is placed in the description.
But using "Resume on focus gain" can raise several questions when seeing this for the first time: Resume what? What is a focus gain?